src/Parser.zig

```diff
@@ -1745,6 +1742,12 @@ fn recordSpec(p: *Parser) Error!Type {
         p.syms.syms.items(.ty)[symbol_index.?] = ty;
     }
 
+    if (!ty.hasIncompleteSize()) {
+        const rl = @import("record_layout.zig");
```

Comment:
This can be imported at the top of the file, and for consistency should be called `RecordLayout.zig`

Reply:
It started as 'RecordLayout' then I happed to read the style guide in the language docs that made me change it

> File names fall into two categories: types and namespaces. If the file (implicitly a struct) has top level fields, it should be named like any other struct with fields using TitleCase. Otherwise, it should use snake_case.

Reply:
Ah right, that makes sense. I do think it can be imported at the top of the file though.

Reply:
fixed